src/docs/forms/components/CalendarInput/examples/07-timePicker.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Combine example with `06-datePicker`, and delete corresponding description file.